### PR TITLE
Update Self balancing bot blog with youtube link

### PR DESCRIPTION
**Pull request by Mani-Sadhasivam**

### CODE REVIEW

File: _posts/2017/08/2017-08-11-self-balancing-bot-using-96boards-part1.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ comments: true
 date: 2017-08-11 01:01:54+00:00
 layout: post
 link: http://www.96boards.org/blog/self-balancing-bot-using-96boards-part1/
-slug: introducing-self-balancing-bot-using-96boards
+slug: self-balancing-bot-using-96boards-part1
```
Comment (shovanuk) [inline _posts/2017/08/2017-08-11-self-balancing-bot-using-96boards-part1.md]: Is this same as https://www.96boards.org/blog/introducing-self-balancing-bot-using-96boards/
One the page is up we should not change the slug, people with the old url will get a 404 page

File: _posts/2017/08/2017-08-11-self-balancing-bot-using-96boards-part1.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ comments: true
...
 featured_image: mpu6050.jpg
 thumbnail_image: mpu6050-thumb.jpg
```
Comment (shovanuk) [inline _posts/2017/08/2017-08-11-self-balancing-bot-using-96boards-part1.md]: Thumbnails are auto generated, only if you have a custom thumbnail add it here. If not delete "thumbnail_image: mpu6050-thumb.jpg"

File: _posts/2017/08/2017-08-11-self-balancing-bot-using-96boards-part1.md [hunk 3]
```diff
@@ -122,7 +122,9 @@ run the C program (server) in background and Python program (client) in foregrou
 
 Source code and detailed instructions can be found in 96Boards [projects repository](https://github.com/96boards/projects/blob/master/imu)
 
-Demonstration video can be found in [96Boards Youtube Channel](https://youtu.be/mswTk_Sliiw)
+Demonstration video:
+
+{% include media.html media_url="https://youtu.be/mswTk_Sliiw" %}
```
Comment (shovanuk) [inline _posts/2017/08/2017-08-11-self-balancing-bot-using-96boards-part1.md]: As per the instruction here 
https://github.com/96boards/website/blob/master/help/blog.md#media-slideshare-includes-etc

Youtube need to be in the following format to be embedded properly
https://www.youtube.com/embed/mswTk_Sliiw

File: _posts/2017/08/2017-08-11-self-balancing-bot-using-96boards-part1.md (hunk 2 above)
Comment (shovanuk) [inline _posts/2017/08/2017-08-11-self-balancing-bot-using-96boards-part1.md]: Instruction on the help document
https://github.com/96boards/website/blob/master/help/blog.md#custom-thumbnails

File: _posts/2017/08/2017-08-11-self-balancing-bot-using-96boards-part1.md (hunk 1 above)
Comment (Mani-Sadhasivam) [inline _posts/2017/08/2017-08-11-self-balancing-bot-using-96boards-part1.md]: `slug: introducing-self-balancing-bot-using-96boards` is for the introductory blog. I've added this to Part-1 blog as a mistake.
So, it should be `slug: self-balancing-bot-using-96boards-part1`

File: _posts/2017/08/2017-08-11-self-balancing-bot-using-96boards-part1.md (hunk 3 above)
Comment (Mani-Sadhasivam) [inline _posts/2017/08/2017-08-11-self-balancing-bot-using-96boards-part1.md]: Fixed

File: _posts/2017/08/2017-08-11-self-balancing-bot-using-96boards-part1.md (hunk 2 above)
Comment (Mani-Sadhasivam) [inline _posts/2017/08/2017-08-11-self-balancing-bot-using-96boards-part1.md]: Fixed

**Pull request merged**